pkg/commands/nuke/nuke.go

```diff
@@ -210,15 +210,22 @@ func execute(c *cli.Context) error { //nolint:funlen,gocyclo
 		region := nuke.NewRegion(regionName, account.ResourceTypeToServiceType, account.NewSession, account.NewConfig)
 
 		// Step 2 - Create the scannerActual object
-		scannerActual := scanner.New(regionName, resourceTypes, &nuke.ListerOpts{
-			Region:    region,
-			AccountID: ptr.String(account.ID()),
-			Logger: logger.WithFields(logrus.Fields{
-				"component": "scanner",
-				"region":    regionName,
-			}),
+		scannerActual, scannerActualErr := scanner.New(&scanner.Config{
+			Owner:         regionName,
+			ResourceTypes: resourceTypes,
+			Opts: &nuke.ListerOpts{
+				Region:    region,
+				AccountID: ptr.String(account.ID()),
+				Logger: logger.WithFields(logrus.Fields{
+					"component": "scanner",
+					"region":    regionName,
+				}),
+			},
+			Logger: logger,
```

Comment:
Need to set `QueueSize` and `ParallelQueries` here

Reply:
:facepalm: yeah would be helpful if I passed the parameters.